components/Tabs/examples/MDCTabBarViewTypicalExampleViewController.m

```diff
@@ -97,6 +102,23 @@ - (void)viewDidLoad {
 
 @end
 
+#pragma mark - MDCTabBarViewDelegate
+@implementation MDCTabBarViewTypicalExampleViewController (MDCTabBarViewDelegate)
+
+- (BOOL)tabBarView:(MDCTabBarView *)tabBarView shouldSelectItem:(nonnull UITabBarItem *)item {
+  // Just to demonstrate prevent selection of the second item.
+  if ([self.tabBar.items indexOfObject:item] == 1) {
```

Comment:
This is just an example so maybe not necessary, but this could just be `return [self.tabBar.items indexOfObject:item] != 1`

Reply:
Done.